src/main/java/edu/harvard/iq/dataverse/authorization/AuthenticationServiceBean.java

```diff
@@ -511,6 +533,69 @@ public AuthenticatedUser lookupUser( String apiToken ) {
         return tkn.getAuthenticatedUser();
     }
     
+    /*
+    getDeleteUserErrorMessages( AuthenticatedUser au )
+    method which checks for reasons that a user may not be deleted
+    -has created dvObjects
+    -has roles
+    -has guestbook records
+    -etc.
+    An empty string is returned if the user is 'deletable'
+    */
+    
+    public String getDeleteUserErrorMessages(AuthenticatedUser au) {
+        String retVal = "";
```

Comment:
it's probably simpler code to add them to a list, then do a join to concatenate them at the end.

Reply:
done